module_cheatsheets/Intro_to_R_Tidyverse_cheatsheet.md

```diff
@@ -0,0 +1,82 @@
+A vignette on the usage of the `dplyr` package can be found [**here**](https://www.rdocumentation.org/packages/dplyr/versions/0.7.8/vignettes/dplyr.Rmd).
+
+| Library/Package                                 | Piece of code                        | What it's called                       | What it does                                                                                                                                                                                      |
+|-------------------------------------------------|--------------------------------------|----------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| dplyr           | [`%>%`](https://www.rdocumentation.org/packages/magrittr/versions/1.5/topics/%25%26gt%3B%25)        | Pipe operator         | Funnels a data.frame through tidyverse operations                                                                                                                                                 |
+| dplyr           | [`filter()`](https://www.rdocumentation.org/packages/dplyr/versions/0.7.8/topics/filter)            | Filter                | Returns a subset of rows matching the conditions of the specified logical argument                                                                                                                |
+| dplyr           | [`arrange()`](https://www.rdocumentation.org/packages/dplyr/versions/0.7.8/topics/arrange)          | Arrange               | Reorders rows in ascending order. arrange(desc()) would reorder rows in descending order.                                                                                                         |
+| dplyr           | [`select()`](https://www.rdocumentation.org/packages/dplyr/versions/0.7.8/topics/select)            | Select                | Isolates columns that match the specified argument                                                                                                                                                |
+| dplyr           | [`mutate()`](https://www.rdocumentation.org/packages/dplyr/versions/0.7.8/topics/mutate)            | Mutate                | Adds a new column that is a function of existing columns                                                                                                                                          |
+| dplyr           | [`summarise()`](https://www.rdocumentation.org/packages/dplyr/versions/0.7.8/topics/summarise)      | Summarise             | Summarises multiple values in an object into a single value. This function can be used with other functions to retrieve a single output value for the grouped values.                                                                                                     |
```

Comment:
May want to also add `summarize` here -- I've had people ask about the spelling and if it is equivalent before.